tools/bazelify_tests/grpc_run_simple_command_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright 2023 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+ARCHIVE_WITH_SUBMODULES="$1"
+shift
+
+# Extract grpc repo archive
+tar -xopf ${ARCHIVE_WITH_SUBMODULES}
+cd grpc
+
+# Remove the override the "do not detect toolchain" setting that was set
+# by the from .bazelrc configuration for the remote build.
```

Comment:
"by the from ..." should maybe just be "by .bazelrc for the remote build"

Reply:
done.